src/private/MDMPresentationTransitionController.m

```diff
@@ -129,4 +144,35 @@ - (void)prepareForTransitionWithSourceViewController:(nullable UIViewController
   }
 }
 
+- (nullable id<UIViewControllerInteractiveTransitioning>) prepareForInteractiveTransition {
```

Comment:
Ideally this method would happen as a result of `prepareForTransition`. We know that the animation callback is invoked first and therefor our transition context will be initialized, so we can return the percent-driven transition controller from our transition context.